src/gui/MainWindow.cpp

```diff
@@ -236,6 +245,8 @@ void MainWindow::showHidden()
 
 void MainWindow::show()
 {
+	setInvisible(false);
+	setEnablements(true);
```

Comment:
Could we have here called the `showWindow` method?

Reply:
No, because `showWindow` calls `MainWindow::show`.

Reply:
I see, and we don't need that here. Thanks for clarifying.